regexes/client/browsers.yml

```diff
@@ -787,6 +787,11 @@
   name: 'Huawei Browser'
   version: '$1'
 
+# Huawei Browser for PC
+- regex: 'HBPC/(\d+[\.\d]+)'
+  name: 'Huawei Browser for PC'
```

Comment:
@sgiehl need help. I'm not sure about the name, usually we try to avoid entering the name for browsers with verbs
Examples: Edge, Chrome, Chrome Mobile, Qwant Mobile.

what about the name -   `Huawei Browser Desktop`  or the current name good?

Reply:
Is the other detected `Huawei Browser` a mobile only one? If so we could rename the other to `Huawei Browser Mobile` and the desktop one simply `Huawei Browser`. That way it would match the other namings